code/processes/dqe.q

```diff
@@ -67,13 +71,21 @@ loadtimer:{[d]
 /- adds today's date to the time from config csv, before loading the queries to the timer
 configtimer:{[]
   t:.dqe.readdqeconfig[.dqe.configcsv;"S**SN"];
-  t:update starttime:(`date$(.z.D,.z.d).dqe.gmttime)+starttime from t;
+  t:update starttime:(`date$(.z.D,.z.d).dqe.utctime)+starttime from t;
   {.dqe.loadtimer[x]}each t
   }
 
 writedownengine:{
   if[0=count .dqe.tosavedown`.dqe.resultstab;:()];
+  dbprocs:exec distinct procname from raze .servers.getservers[`proctype;;()!();0b;1b]each`hdb`dqedb`dqcdb;  // Get a list of all databases.
+  restemp1:select from .dqe.resultstab where procs in dbprocs;
```

Comment:
Can you explain what the purpose is of splitting these up and then saving separately?

I'm guessing this is to do with getting the right date but how is that being determined from dbprocs?  Needs to be more commenting around this for those coming behind.

Reply:
When procs in resultstab is a database, that means that data is from yesterday. For example, today's(08.27) would be looking at hdb's data from 08.26. However, when procs arent databases, lets say its looking at symfiles, then it should save down as todays date. Thats why we split it up